eng/pipelines/publish-release.yml

```diff
@@ -0,0 +1,143 @@
+trigger: none
+pr: none
+
+variables:
+  NodeVersion: '12.x'
+  goTestPath: '$(system.defaultWorkingDirectory)/test'
+  GOBIN: '$(system.defaultWorkingDirectory)/bin/go'
+
+pool:
+  vmImage: 'ubuntu-18.04'
+
+variables:
+  NodeVersion: '12.x'
+  TestFolder: '$(Build.SourcesDirectory)/test/'
+
+steps:
+  - task: NodeTool@0
+    displayName: 'Install Node.js $(NodeVersion)'
+    inputs:
+      versionSpec: '$(NodeVersion)'
+
+  - task: UsePythonVersion@0
```

Comment:
I assume all the Python stuff can be removed yes?  Lines 22 through 51.

Reply:
Ouch, didn't mean to leave that in there.  Thanks for catching it!